src/components/CJobRow.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <tr class="border-b border-gray-200 hover:bg-blue-100">
+    <td class="border px-4 py-2">
+      <pre class="text-xs whitespace-normal ml-2 "> {{ actorName }}</pre>
```

Comment:
You don't need all these pre. Why use them ? 
also add text-xs class to tr. It will be simpler